README.md

```diff
@@ -4,57 +4,66 @@ ULI Service with Backend Ingest Process and Matching Service for User Deduplicat
 This work uses the [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) probabilistic matching algorithm for its search implementation, with a set of weights similar to those outlined in the [Sandbox document](https://docs.google.com/document/d/10YFyqw9hIwBXPjpX6yGFQoJUHWpL5M33sVHp5sEjX-Y/edit?usp=sharing) discussed in the RESO ULI Subgroup previously.
 
 
-This repository contains the following items:
-* A [template spreadsheet](https://github.com/RESOStandards/uli-service/blob/main/ULI%20-%20Data%20Pilot%20Template.xlsx?raw=true), which will allow you to ingest data into the ULI Pilot backend using your local data. 
-* A [`docker-compose.yml` file](https://github.com/RESOStandards/uli-service/blob/main/docker-compose.yml) that will start the Elastic backend for you.
-* A [text file](https://github.com/RESOStandards/uli-service/blob/main/uli-pilot-ingest.txt) containing an Elastic Ingest Pipeline for the template spreadsheet.
-* A file containing a [ULI Pilot Search query](https://github.com/RESOStandards/uli-service/blob/main/uli-ranking-formula.json), which you can fill in values for once you have ingested data.
-
-# Getting Started
-
-## 1. Preparing ULI Pilot Data for Ingest
-
-The rest of the steps in this README are optional, but to participate in the ULI Pilot there needs to be an initial seed file created from the Member and Office data in the organization at that time. 
-
-A [Template Spreadsheet](https://github.com/RESOStandards/uli-service/blob/main/ULI%20-%20Data%20Pilot%20Template.xlsx?raw=true) has been provided for your convenience. 
-
-Please fill in the fields on the "Merged" tab of the spreadsheet and send the sheet to [RESO Development](mailto:dev@reso.org). Feel free to reach out with any additional questions. 
-
-For those who are running the server locally, please proceed to the next step after filling their data into the Merged sheet.
+### Setting up project for the first time
 
+Make sure you have [Docker and Docker Compose installed](https://docs.docker.com/compose/install/). The Windows and MacOs installers bundle them both together. The referenced guide has instructions for how to get started with both. 
 
-## 2. Starting the Elastic Backend
+- Change permissions to a sub-folder `sudo chown -R 1000:1000 ./docker-data-volumes/elasticsearch`
+- Run your project with the command `docker-compose up`
```

Comment:
Didn't work for me on Linux 5.15.8, Docker version 20.10.7, build 20.10.7-0ubuntu5.1, and Ubuntu 21.10.

Got the following error:

```
docker-compose up
Starting uli_elasticsearch ... error

ERROR: for uli_elasticsearch  Cannot start service elasticsearch: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/home/jdarnell/work/reso/github/uli-service/elasticsearch.yml" to rootfs at "/usr/share/elasticsearch/config/elasticsearch.yml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

ERROR: for elasticsearch  Cannot start service elasticsearch: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/home/jdarnell/work/reso/github/uli-service/elasticsearch.yml" to rootfs at "/usr/share/elasticsearch/config/elasticsearch.yml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.
```

Stopping here for now until this works.

Reply:
I have:
Docker version 20.10.8
docker-compose version 1.29.2

You may need to upgrade.  Using version: "3.9" for docker-compose needs the bleeding edge of these programs.
I'll update the documentation.

This is exactly how ther reso-certification project works.  Are you using a different environment as your development environment for that project?

Reply:
Ah, I see a file was also missing from the repo.  Now that I've pruned the .p12 files, I can include the needed file.